zebra-network/src/peer/handshake.rs

```diff
@@ -380,7 +380,7 @@ where
                         if msg.is_ok() {
                             // XXX add a dimension tagging message metrics by type
```

Comment:
`zcashd` implements this dimension as a `command` label; it should probably be added in this PR, so the metric is consistent across nodes.